templates/download/shared/_server_weekly_news.html

```diff
@@ -0,0 +1,73 @@
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-12">
+      <h3>Get Ubuntu Server weekly news</h3>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-7">
+      <p>Subscribe to the Ubuntu Server weekly brief to receive:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Security briefs</li>
+        <li class="p-list__item is-ticked">Release announcements</li>
+        <li class="p-list__item is-ticked">General technology updates</li>
+        <li class="p-list__item is-ticked">Tips and tutorials</li>
+      </ul>
+      <h4>Need a command-line refresher?</h4>
+      <p>By subscribing, you also get the "Ubuntu Server CLI pro tips 2019" cheat sheet: learn how to use efficiently the command-line and get started with DevOps &mdash; from basic file management to deploying Kubernetes and OpenStack.</p>
+      <p>Quick tips for the command line and getting started with DevOps &mdash; from basic file management to deploying Kubernetes and OpenStack.</p>
+      <div class="p-card--highlighted" style="display: inline-block;">
+        <img src="https://assets.ubuntu.com/v1/83facbfb-Screenshot+from+2019-06-05+10-27-40.png" alt="" width="440" />
+      </div>
+    </div>
+
+    <div class="col-4 col-start-large-9">
+      <!-- MARKETO FORM -->
+      <form action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_1917" class="marketo-form">
+        <fieldset style="border: 0; margin: 0; padding: 0;">
+          <ul class="p-list">
+            <li class="mktFormReq mktField p-list__item">
+              <label for="firstName" class="mktoLabel">First name:</label>
+              <input required  id="firstName" name="FirstName" maxlength="255" type="text" class="mktoField mktoRequired" >
+            </li>
+            <li class="mktFormReq mktField p-list__item">
+              <label for="lastName" class="mktoLabel">Last name:</label>
+              <input required  id="lastName" name="LastName" maxlength="255" type="text" class="mktoField mktoRequired">
+            </li>
+            <li class="mktFormReq mktField p-list__item">
+              <label for="company" class="mktoLabel">Company Name:</label>
+              <input required  id="company" name="Company" maxlength="255" type="text" class="mktoField mktoRequired">
+            </li>
+            <li class="mktFormReq mktField p-list__item">
+              <label for="email" class="mktoLabel ">Work email:</label>
+              <input required  id="email" name="Email" maxlength="255" type="email" class="mktoField mktoEmailField mktoRequired">
+            </li>
+            <li class="mktFormReq mktField p-list__item">
```

Comment:
Phone number can be dropped. Form is: "First name, Last name, Company name, Work email"